docs/custom.css

```diff
@@ -4,6 +4,9 @@ code { font-size: 90% !important }
 h1, h2, h3, h4, h5 { margin: 1rem 0; width: 100%; }
 ul li { margin-left: 1rem }
 
+.sticky { position: sticky }
```

Comment:
Can we use `position-sticky` instead of this?

https://primer.style/css/utilities/layout#sticky

Reply:
Don't know why I couldn't find this when I looked :blush: